.github/CONTRIBUTING-en-US.md

```diff
@@ -1,6 +1,8 @@
+**English (US)** | [Português (BR)](CONTRIBUTING.md)
+
 # Contributing
 
-*Please note that this project is released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md). By participating in this project, you agree to abide by its terms.*
+> To contribute to Querido Diário, read the [Guide for Contributors](https://github.com/okfn-brasil/querido-diario-comunidade/blob/main/.github/CONTRIBUTING-en-US.md#contributing). This guide provides general information such as our code-of-conduct, the project's list of repositories and the first steps you can take.
```

Comment:
Tava pensando aqui... por enquanto acho que seria melhor pegar as partes atualizadas em português e colocar aqui do que manter o antigo.

Colocar um 
```plain
This section needs translation starting here:
```
ou algo do tipo em cada seção ajudaria a demarcar.

Não fizemos isso no querido-diario-comunidade mas aqui tá incomodando muito mais porque tá trazendo informação muito desatualizada

Reply:
Tinha escrito ali na msg do PR que ia abrir issue de tradução imediatamente depois de trabalharmos aqui no PR. Não são ações conflitantes, mas acho que deve ser resolvido logo. Oq acha?

Reply:
Não são conflitantes e é ótimo ter as issues depois dessa atualização. Mas enquanto os trechos estiverem sem tradução seria melhor não manter documentação obsoleta. Vai ficar mais fácil pra traduzir e seria melhor não conseguir ler do que ler coisa errada.

Talvez nem seja necessária a sugestão de mensagem pra demarcar o início de onde seria necessário tradução pra não poluir demais. Talvez só no início da página dizendo que trechos dessa página ainda precisam de tradução. O que acha?